### ISSUE TITLE
Error when creating a concept

### ISSUE THREAD

**Issue by NCIEVS** [NCIEVS/nci-protege5#255]
I was trying to create a new concept and when I hit save an error message appeared saying something needed to be Committed before I could save (sorry I didn’t think to capture this). Then when I went to Server>Commit, I got an error message saying I needed to Update first. When I did Server>Update, it said Local Copy is Up-To-Date but I kept getting the same issues when trying to use the Save button or Server>Commit. So I had to cancel and since it was a new, never-saved concept, I had to start the create from scratch.

If this happens again I will try to do a better job capturing the error messages. (There was nothing in the console, let me know if I should do a thread-dump next time.)

**Comment by NCIEVS** [comment]
In a related item I was trying to save a cloned concept when that message appeared. I hit OK to close the window, but the two concepts in the clone pane disappeared. I went to Server>Commit and got a message saying I needed to update, so then
I hit Update and was told the local copy was up-to-date. I could see my edits in the uncommitted changes log, but it took two more tries to Commit the changes. (I think if other editors run into this, they will be very confused because we did not review or anticipate needing to Update or Commit to Save.)
